ui/list_multiselect_widget.py

```diff
@@ -141,6 +160,8 @@ def _setupUI(self):
         self.deselect_btn.setToolTip('Remove the selected items')
         self.select_all_btn.setToolTip('Add all')
         self.deselect_all_btn.setToolTip('Remove all')
+        spacer_label = QtGui.QLabel()
+        self.buttons_vertical_layout.addWidget(spacer_label)
```

Comment:
It was a dirty way to make its height exactly the same as the "Unselected" and "Selected" labels, but I agree it would be cleaner to use a spacer. I am going to fix it.